lib/opensusebasetest.pm

```diff
@@ -914,10 +914,10 @@ sub wait_boot {
         $self->handle_pxeboot(bootloader_time => $bootloader_time, pxemenu => 'pxe-custom-kernel', pxeselect => 'pxe-custom-kernel-selected');
     }
     # When no bounce back on power KVM, we need skip bootloader process and go ahead when 'displaymanager' matched.
-    elsif (get_var('OFW') && (check_screen('displaymanager', 5))) {
+    elsif (get_var('OFW') && (check_screen('displaymanager', 10))) {
```

Comment:
Why increase this timeout on ppc64le?

Reply:
To make understand that 5sec timeout are to few to allow this screen to be detected,it'll take too long discussions: restored to oiginal.